views/aggregate-block-metadata.json

```diff
@@ -2083,7 +2083,7 @@
         "slug": "ecs-task",
         "logo_url": "https://cdn.sanity.io/images/3ugk85nk/production/d74b16fe84ce626345adf235a47008fea2869a60-225x225.png",
         "documentation_url": "https://prefecthq.github.io/prefect-aws/ecs/#prefect_aws.ecs.ECSTask",
-        "description": "Run a command as an ECS task. This block is part of the prefect-aws collection. Install prefect-aws with `pip install prefect-aws` to use this block.",
+        "description": "DEPRECATION WARNING: This block is deprecated along with Agents and all other Infrastructure blocks. It will be removed in prefect v3. Run a command as an ECS task. This block is part of the prefect-aws collection. Install prefect-aws with `pip install prefect-aws` to use this block.",
```

Comment:
nit, for this and below
```suggestion
        "description": "DEPRECATION WARNING: This block is deprecated along with Agents and all other Infrastructure blocks. It will be removed in `prefect>=3.0.0`. Run a command as an ECS task. This block is part of the prefect-aws collection. Install prefect-aws with `pip install prefect-aws` to use this block.",
```

Reply:
and so it does 😢 

brb

Reply:
oh jeez, i see. i guess this: prefect>=3.0.0 works for me just as well

Reply:
I think the ticks do it. 

![image](https://github.com/PrefectHQ/prefect-collection-registry/assets/22418768/2dd3683d-6b07-4281-8bb9-031f350d0081)